### PR TITLE
[Amica.com] New ruleset

### PR DESCRIPTION
**Pull request by jeremyn**

### CODE REVIEW

File: src/chrome/content/rules/Amica.com.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<ruleset name="Amica.com">
+	<target host="www.amica.com" />
+	<target host="m.amica.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Amica.com.xml]: careers.amica.com
securemail.amica.com

Please also add a comment for the timeouts on futureleaders.amica.com and amica.com (without www).

**Comment by jeremyn** [comment]
I've added careers.amica.com and securemail.amica.com to the ruleset, and a comment for ^ and futureleaders.amica.com.

**Comment by J0WI** [comment]
Okay, the cert for securemail.amica.com just expired right now.

**Comment by jeremyn** [comment]
Let's give them a week or so to renew the certificate, and if they don't then I'll move the securemail domain to the comment.

**Comment by jeremyn** [comment]
The securemail domain certificate has been renewed, so you should be able to merge this pull request now.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**